packages/badge/README.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+Badges show a small amount of color-categorized metadata. They're ideal for getting a user's attention.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/badge?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/badge)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/badge?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/badge)
+
+```
+yarn add @spectrum-web-components/badge
+```
+
+Import the side effectful registration of `<sp-badge>` via:
+
+```
+import '@spectrum-web-components/badge/sp-badge.js';
+```
+
+When looking to leverage the `Badge` base class as a type and/or for extension purposes, do so via:
+
+```
+import { Badge } from '@spectrum-web-components/badge';
+```
+
+## Example
```

Comment:
For element with t-shirt sizing, we use an `<sp-tabs>` patters to feature all of the sizes here. Can you review https://spectrum-css--spectrum-web-components.netlify.app/components/action-button/#sizes for that?